src/index/coinstatsindex.cpp

```diff
@@ -24,10 +24,10 @@ struct DBVal {
     uint64_t bogo_size;
     CAmount total_amount;
     CAmount total_subsidy;
-    CAmount block_unspendable_amount;
-    CAmount block_prevout_spent_amount;
-    CAmount block_new_outputs_ex_coinbase_amount;
-    CAmount block_coinbase_amount;
+    CAmount total_unspendable_amount;
+    CAmount total_prevout_spent_amount;
+    CAmount total_new_outputs_ex_coinbase_amount;
+    CAmount total_coinbase_amount;
```

Comment:
`unspendables` are also "total", but don't mention it. Might be good to remove the "total_" here, or add it to unspendables?

Reply:
Dropped the `total_` everywhere except for the `m_total_amount`, it seemed right to keep it in this case since it's closed to the corresponding legacy name `nTotalAmount` that is also still in use.

Reply:
Dropped the `total_` everywhere except for the `m_total_amount`, it seemed right to keep it in this case since it's closed to the corresponding legacy name `nTotalAmount` that is also still in use.